lib/tests/fixtures.hpp.in

```diff
@@ -3,6 +3,9 @@
 #include <string>
 #include <functional>
 
+#define JAVA_EXECUTABLE "@Java_JAVA_EXECUTABLE@"
```

Comment:
Weird, I get no errors, it just defines JAVA_EXECUTABLE to empty string :/ What error are you seeing?